### PR TITLE
Package pyvisfile

### PR DESCRIPTION
**Pull request by inducer**

### CODE REVIEW

**Comment by inducer** [comment]
Ready to review from my end.

File: recipes/pyvisfile/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - pytools
+        - six
```
Comment (jakirkham) [inline recipes/pyvisfile/meta.yaml]: Please cut anything not needed at build time from the build dependencies (probably just `python` and `setuptools`).

**Comment by jakirkham** [comment]
Also LGTM. Minor comments to resolve and can be merged.

File: recipes/pyvisfile/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    imports:
+        - {{ name }}
+
+about:
+    home: http://pypi.python.org/pypi/pyvisfile
+    dev_url: http://github.com/inducer/pyvisfile
+    doc_url: https://documen.tician.de/pyvisfile
+    license: MIT
+    license_file: LICENSE
+    summary: Large-scale Visualization Data Storage
```
Comment (inducer) [inline recipes/pyvisfile/meta.yaml]: Done.

**Comment by inducer** [comment]
Thanks for the review!

File: recipes/pyvisfile/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    description: |
+
```
Comment (jakirkham) [inline recipes/pyvisfile/meta.yaml]: Might lose this extra whitespace, but not a blocker.

**Comment by jakirkham** [comment]
Thanks @inducer.

**Pull request merged**